src/types/AdInfo.ts

```diff
@@ -48,6 +48,14 @@ export type AdInfo = {
      * The native ad info.
      */
     nativeAd?: AdNativeInfo | null;
+
+    /**
+     * The size of AdView for adaptive banners.
+     */
```

Comment:
maybe we can just copy our native API docs in a similar manner:

<img width="555" alt="image" src="https://github.com/AppLovin/AppLovin-MAX-React-Native/assets/5104410/3493f3d0-42f7-47e8-95bb-b5d9f1065db5">

Reply:
Copied.